tests/plugins/tableselection/tableselection.js

```diff
@@ -20,7 +20,7 @@
 	}
 
 	var tests = {
-		// (#258, #tp2247)
+		// (http://dev.ckeditor.com/ticket/258, #tp2247)
```

Comment:
It seems like this ticket number is from unknown source… It does not fit to trac or our GH.